### PR TITLE
Add a document for moloco-sdk-native, and seperate documents of the modular SDK. 

### PR DESCRIPTION
**Pull request by soohan-ahn**

### CODE REVIEW

File: android-sdk/BANNER.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+Initialization requires `AdUnitID` provided by Moloco. Please contact [Moloco](mailto:support@molocoads.com) if you need `AdUnit ID`.
+
+```java
+String adUnitId = "your_ad_unit_id";
+mMolocoView.setAdUnitId(adUnitId);
+mMolocoView.loadAd();
+mMolocoView.setBannerAdListener(this);
+```
+
+Now you are to ready use **Moloco Android SDK** for Android devices!
```
Comment (myunggeun-moloco) [inline android-sdk/BANNER.md]: Typo: you are ready to use (here and below)

File: android-sdk/BANNER.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+Initialize SdkConfiguration with a LogLevel (INFO or DEBUG).
+
+```java
+LogLevel logLevel = INFO; // or DEBUG
+final SdkConfiguration sdkConfiguration = new SdkConfiguration(logLevel);
+Moloco.initializeSdk(this, sdkConfiguration);
+```
+
+###  Load an Ad with an ad unit id.
+Initialization requires `AdUnitID` provided by Moloco. Please contact [Moloco](mailto:support@molocoads.com) if you need `AdUnit ID`.
```
Comment (myunggeun-moloco) [inline android-sdk/BANNER.md]: Let's keep it consistent. ad unit id / AdUnitID / AdUnit ID. (here and below)

File: android-sdk/BANNER.md (hunk 1 above)
Comment (soohan-ahn) [reply android-sdk/BANNER.md]: Thank you for your check.
I fixed it on here - 2a26f08 .

File: android-sdk/BANNER.md (hunk 2 above)
Comment (soohan-ahn) [reply android-sdk/BANNER.md]: Thank you for your check.
I fixed it on here - a02ac74 .

**Comment by hyeonseo** [comment]
Looks good to me

**Review by myunggeun-moloco — APPROVED** [review]
LGTM

**Pull request merged**